docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md

```diff
@@ -0,0 +1,135 @@
+# Job manager and the JOB statements
+
+The long-term tasks running at the storage layer are called jobs. For example, there are jobs for `COMPACT`, `FLUSH`, and `STATS`. These jobs can be time-consuming if the data size in the graph space is large. The job manager helps you run, show, stop and recover the jobs.
+
+## SUBMIT JOB COMPACT
+
+The `SUBMIT JOB COMPACT` statement triggers the long-term RocksDB compact operation.
+
+```ngql
+nebula> SUBMIT JOB COMPACT;
++------------+
+| New Job Id |
++------------+
+| 40         |
++------------+
+```
+
+For more information about compact configuration, see Storage Service configuration (doc TODO).
+
+## SUBMIT JOB FLUSH
+
+The `SUBMIT JOB FLUSH` statement writes the RocksDB memfile in memory to the hard disk.
```

Comment:
add an eg here?

Reply:
It returns almost the same thing with `SUBMIT JOB COMPACT`, so I think it is not needed.

Reply:
Give an example. I think.